examples/staticcheck/BUILD.bazel

```diff
@@ -0,0 +1,33 @@
+load("@bazel_gazelle//:def.bzl", "gazelle")
+load("@com_github_sluongng_nogo_analyzer//staticcheck:def.bzl", "staticcheck_analyzers")
+load("@com_github_sluongng_nogo_analyzer//:def.bzl", "nogo_config")
+load("@io_bazel_rules_go//go:def.bzl", "TOOLS_NOGO", "nogo")
+
+# gazelle:prefix github.com/sluongng/nogo-analyzer/examples/staticcheck
+gazelle(name = "gazelle")
+
+gazelle(
+    name = "deps",
+    args = [
+        "-from_file=go.mod",
+        "-to_macro=go_deps.bzl%go_deps",
+        "-prune",
+    ],
+    command = "update-repos",
+)
+
+load(":staticcheck.bzl", "STATICCHECK_ANALYZERS", "STATICCHECK_OVERRIDE")
```

Comment:
`load` statement should be at the top of the file according to buildifier lint

Reply:
Moved.